README.md

```diff
@@ -1,13 +1,11 @@
 # Cluedo - Mystery Randomizer
```

Comment:
Mats, really good job with your Cluedo project this week! I know your circumstances haven't been great and you have done really well in solving this challenging task even though you have been sick! I think you did a great job with the images in the respective deck - as I found it really hard to get them to size properly in each card (and not stick outside the cards). 👍🥇 Great work😄